src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const handleTabSelected = id => {
+    if (id !== activeTabId) {
+      onTabSelected(id);
+    }
+  };
+
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  return (
+    <>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={cn({ 'is-active': tab.id === activeTab.id })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={e => {
+                  e.preventDefault();
+                  if (tab.id !== activeTabId) {
+                    handleTabSelected(tab.id);
+                  }
```

Comment:
Still not fixed, move this logic to the helper function and use it here 
<img width="905" alt="image" src="https://github.com/user-attachments/assets/8f55de23-82d5-47ad-878e-dfc660530ff4">